assets/css/style.css

```diff
@@ -178,7 +178,19 @@ h2 {
     color: #ccc;
 }
 
+.txt-type {
+    color: rgba(255, 255, 0, 0.8);
+}
+
 /* Cursor */
-.txt-type > .txt {
-    border-right: 0.1rem solid #ccc;
+.txt-type::after {
+    color: #fff;
+    content: "_";
+    animation: animate-cursor 1s infinite;
+}

```

Comment:
Here also the same color as that of the text.

Reply:
@Rocker2102

Reply:
**`#FF5722`** looks the best.

- [ ] ![color-1](https://user-images.githubusercontent.com/43738236/94605289-87c3ed80-02b6-11eb-84bd-6a542431f94a.png)
- [ ] ![color-2](https://user-images.githubusercontent.com/43738236/94605294-88f51a80-02b6-11eb-89c6-101de4bb0210.png)
- [x] ![color-3](https://user-images.githubusercontent.com/43738236/94605296-88f51a80-02b6-11eb-965b-0a36893eac02.png)